saos-webapp/src/main/webapp/WEB-INF/view/details/judgmentDetails.jsp

```diff
@@ -259,12 +294,16 @@ $(document).ready(function() {
 			</c:if>
 			
 			<c:if test="${!empty judgment.keywords}" >
+			    <spring:message code="judgmentDetails.linkTooltip.keywords" var="keywordsLinkTooltip" />
 				<li>
 					<div class="" >
 						<div class="label-title" ><spring:message code="judgment.keywords" />:</div>
 						<div class="desc" >
 							<c:forEach items="${judgment.keywords}" var="keyword" varStatus="status">
-								<c:out value="${keyword.phrase}" /><c:if test="${!status.last}">,</c:if>
+							<a href="${contextPath}/search?courtCriteria.courtType=COMMON&amp;keywords=${keyword.phrase}" data-toggle="tooltip" title="${keywordsLinkTooltip}" >
```

Comment:
If we would have keywords not only in common court judgments, then `?courtCriteria.courtType=${judgment.courtType}` would be more generic